app/views/admin/users/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<%= form_for [:admin, @user] do |f| %>
```

Comment:
@AliSchlereth @iandouglas I am stuck here. I can't get this form to work. Without the :admin it routes the PUT request to the Users controller, but with the :admin it loads the page twice. I stuck a binding.pry in on line 1 and the first time the page loads it has the @user object. The second time it loads @user is nil and the form breaks. This is from the admin_can_manage_users_spec. I will also mark the line where the problem happens in the test.

Reply:
@anon0mys This sounds to me like a save is failing and therefore a render is getting hit. Keep in ming that you only have access to instance variables within that controller action since each time you make a request you are on a new instance of the controller.  If you are routing to your update, and the update doesn't trigger the if statement, render is run but you only have access to the variables in that update method.